README.md

```diff
@@ -30,3 +30,22 @@ helm install \
 ```
 
 Assuming a values file ``myvalues.yml`` was created.
+
+## Elastic Search recommendations
+
+GPP-zoeken requires Elastic Search (v8.x, tested with 8.17) to be deployed to function
+properly. We recommend installing Elastic Search in your cluster using their
+[operator](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-deploy-eck.html).
```

Comment:
Maybe we should link to the helm installation specifically:
https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-install-helm.html